spec/serializers/participant_serializer_spec.rb

```diff
@@ -4,20 +4,30 @@
 
 RSpec.describe ParticipantSerializer do
   describe "serialization" do
-    let(:mentor) { create(:participant_profile, :mentor).user }
+    let(:mentor_profile) { create(:participant_profile, :mentor) }
+    let(:mentor) { mentor_profile.user }
     let(:ect_profile) { create(:participant_profile, :ect, mentor_profile: mentor.mentor_profile) }
     let(:ect) { ect_profile.user }
     let(:ect_cohort) { ect.early_career_teacher_profile.cohort }
     let(:mentor_cohort) { mentor.mentor_profile.cohort }
 
-    it "outputs correctly formatted serialized Mentors" do
-      expected_json_string = "{\"data\":{\"id\":\"#{mentor.id}\",\"type\":\"participant\",\"attributes\":{\"email\":\"#{mentor.email}\",\"full_name\":\"#{mentor.full_name}\",\"mentor_id\":null,\"school_urn\":\"#{mentor.mentor_profile.school.urn}\",\"participant_type\":\"mentor\",\"cohort\":#{mentor_cohort.start_year},\"status\":\"active\",\"teacher_reference_number\":\"#{mentor.teacher_profile.trn}\",\"teacher_reference_number_validated\":true,\"eligible_for_funding\":null,\"pupil_premium_uplift\":null,\"sparsity_uplift\":null}}}"
-      expect(ParticipantSerializer.new(mentor).serializable_hash.to_json).to eq expected_json_string
-    end
+    context "when the participant is eligible for funding" do
+      before do
+        eligibility = ECFParticipantEligibility.create!(participant_profile: ect_profile)
+        eligibility.update!(status: :eligible)
```

Comment:
You're missing lots of sugar, as this is an enum you can do this instead:
```suggestion
        eligibility.eligible_status!
```